lib/modules/manager/bun/index.ts

```diff
@@ -4,7 +4,7 @@ import { NpmDatasource } from '../../datasource/npm';
 
 export { updateArtifacts } from './artifacts';
 export { extractAllPackageFiles } from './extract';
-export { getRangeStrategy, updateDependency } from '../npm';
+export { getRangeStrategy, updateDependency, detectGlobalConfig } from '../npm';
```

Comment:
If this isn't essential for this PR, please move it to a separate discussion. I'm not sure this export even achieves anything